app/Filament/Terrain/Pages/ScanChantierEquipmentPage.php

```
@@ -0,0 +1,249 @@
+        ]);
+    }
+
+    public function form(Schema $form): Schema
+    {
+        return $form
+            ->schema([
+                Section::make('Scan du Matériel')
+                    ->schema([
+                        Select::make('chantier_id')
+                            ->label('Chantier')
+                            ->options(fn () => Chantier::forEmployee(Auth::user()->salarie ?? Auth::user()->employee ?? null)
```

PHPStan (2.2.8)

[error] Call to an undefined static method App\Models\Chantiers\Chantier::forEmployee().


(staticMethod.notFound)